mautic/mautic
# Blank theme if no theme exists

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 




#### Description:
If the template is removed for email or landing page, the blank theme should be selected.



#### Steps to test this PR:
1. Remove template from the code
2. Check if blank template is selected




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/10154"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
